internal/oid/oid.go

```diff
@@ -72,10 +112,29 @@ func (AzureTrustedLaunch) OID() asn1.ObjectIdentifier {
 	return asn1.ObjectIdentifier{1, 3, 9900, 4, 2}
 }
 
+// String returns the string representation of the OID.
+func (AzureTrustedLaunch) String() string {
+	return azureTL
+}
+
 // QEMU holds the QEMU OID.
 type QEMU struct{}
 
 // OID returns the struct's object identifier.
 func (QEMU) OID() asn1.ObjectIdentifier {
 	return asn1.ObjectIdentifier{1, 3, 9900, 5, 1}
 }
+
+// String returns the string representation of the OID.
+func (QEMU) String() string {
+	return qemuVTPM
+}
+
+const (
+	dummy       = "dummy"
+	awsNitroTPM = "aws-nitrotpm"
+	gcpSEVES    = "gcp-seves"
+	azureSEVSNP = "azure-sevsnp"
+	azureTL     = "azure-trustedlaunch"
```

Comment:
```suggestion
	awsNitroTPM = "aws-nitro-tpm"
	gcpSEVES    = "gcp-se-ves"
	azureSEVSNP = "azure-sev-snp"
	azureTL     = "azure-trustedlaunch"
```
looks a bit cleaner imo

Reply:
I was going for `<CSP>-<SUBVARIANT>` but this does look cleaner.